FullEvents/LoginViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  LoginViewController.swift
+//  FullEvents
+//
+//  Created by user on 09/10/17.
+//  Copyright © 2017 user. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+import SwiftyJSON
+import FullAuthIOSClient
+import SafariServices
+
+class LoginViewController: UIViewController, SFSafariViewControllerDelegate, UIWebViewDelegate{
+    
+    var svc: SFSafariViewController?
+    
+    
+    let urlString = "https://access.anywhereworks.com/o/oauth2/auth?response_type=code&client_id=29354-4dfad15c1bcc7b057adb96651882db0f&redirect_uri=com.fullCreative.FullEvents:/oauth2callback&scope=awapis.users.read&access_type=offline&approval_prompt=force"
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        NotificationCenter.default.addObserver(self, selector: #selector(LoginViewController.accessTokenMethod(_: )), name: NSNotification.Name(rawValue: "Dismiss safari"), object: nil)
```

Comment:
What is accessTokenMethod() ? method name doesnot convey what is handled in that method.